libs/core/src/lib/menu/menu.component.ts

```diff
@@ -205,6 +212,7 @@ export class MenuComponent
             this._popoverService.open();
         }
         this._changeDetectorRef.markForCheck();
+        this.menuItems.first.elementRef.nativeElement.focus();
```

Comment:
This doesn't seem to be working


https://user-images.githubusercontent.com/2471874/200667299-52207cbc-bef0-4668-a471-2af26eab1908.mov

Reply:
oh yes, sorry, I forgot to mention here. 

So, the issue was, when `focusAutoCapture` is `true`, and `popover` opens, it focuses first `tabbable` element which is scrollbar (so not focusing first item). To solve this, I had added this commit (5ed73078a9cf25024fc22daf430cb33a74613b2c) but now to handle `tabindex` of scrollbar we have separate issue https://github.com/SAP/fundamental-ngx/issues/8929, so I reverted this commit.

But all these still were causing the test fail (`focus first element on open`), so I added this line to just pass the test.

Let me know your thought on this.
@mikerodonnell89 @InnaAtanasova @platon-rov